CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [Unreleased](https://github.com/denysdovhan/spaceship-prompt/tree/HEAD)
+
+[Full Changelog](https://github.com/denysdovhan/spaceship-prompt/compare/v3.0.2...HEAD)
+
+**Fixed bugs:**
+
+- \(\#q\) glob qualifier doesn't work for zsh 5.0.5 [\#190](https://github.com/denysdovhan/spaceship-prompt/issues/190)
```

Comment:
Issues like this should probably be labeled as `wontfix` and be excluded from the changelog IMO